.github/pkl-workflows/helpers/Lint.pkl

```diff
@@ -0,0 +1,69 @@
+module lint
+
+import "../GithubAction/GithubAction.pkl" as gha
+import "Steps.pkl"
+import "Common.pkl"
+
+const actionVerifyTODO = "nirinchev/verify-todo@ffa352ac028b4cdc8bc626471d33aa341b3ab7c9"
+
+function lint(): gha.Job = new {
+  name = "Verify TODOs"
+  `runs-on` = new gha.UbuntuLatest{}
+  steps {
+    Steps.checkoutWithoutMatchers(false)
+    new gha.Step {
```

Comment:
I think we can leave the `gha.Step` here out, right? Mostly a question more than a comment.
I can see we leave it out in `verifyNamespaces`